recipes/verr/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
No need to include this file manually since it's present in the PyPI tarball. Please remove.